src/views/Home.vue

```diff
@@ -1,68 +1,120 @@
+    <h1>LOGO</h1>
+    <ion-title>Purchase Order</ion-title>
+  </ion-toolbar>
     <ion-content :fullscreen="true">
-      <ion-header collapse="condense">
-        <ion-toolbar>
-          <ion-title size="large">Blank</ion-title>
-        </ion-toolbar>
-      </ion-header>
-    
-      <div id="container">
-        <strong>Ready to create an app?</strong>
-        <p>Start with Ionic <a target="_blank" rel="noopener noreferrer" href="https://ionicframework.com/docs/components">UI Components</a></p>
+      <div class="filterButton">
+        <ion-button>Pending</ion-button>
+        <ion-button>Approve</ion-button>
+        <ion-button>Declined</ion-button>
       </div>
+      <ion-grid class="tableTitle">
+        <ion-row>
+          <ion-col> PO Number </ion-col>
+          <ion-col> PO Description </ion-col>
+          <ion-col> Request Date </ion-col>
+          <ion-col> Status </ion-col>
+          <ion-col> Action </ion-col>
+        </ion-row>
+      </ion-grid>
+      <ion-grid>
+        <ion-row>
+          <ion-col> {{ order[0].purchaseOrderNumber }} </ion-col>
+          <ion-col> {{order[0].description}} </ion-col>
+          <ion-col> {{order[0].date}} </ion-col>
+          <ion-col> {{order[0].status}} </ion-col>
+          <ion-col> {{order[0].action}} </ion-col>
+        </ion-row>
+        <ion-row>
+          <ion-col> PO-0001 </ion-col>
+          <ion-col> sample Description </ion-col>
+          <ion-col> July 22, 2021 </ion-col>
+          <ion-col> Pending </ion-col>
+          <ion-col> Review </ion-col>
+        </ion-row>
+        <ion-row>
+          <ion-col> PO-0001 </ion-col>
+          <ion-col> sample Description </ion-col>
+          <ion-col> July 22, 2021 </ion-col>
+          <ion-col> Pending </ion-col>
+          <ion-col> Review </ion-col>
+        </ion-row>
+        <ion-row>
+          <ion-col> PO-0001 </ion-col>
+          <ion-col> sample Description </ion-col>
+          <ion-col> July 22, 2021 </ion-col>
+          <ion-col> Pending </ion-col>
+          <ion-col> Review </ion-col>
+        </ion-row>
+      </ion-grid>
+      <ion-button id="addButton">add Icon</ion-button>
     </ion-content>
   </ion-page>
 </template>
 
 <script lang="ts">
-import { IonContent, IonHeader, IonPage, IonTitle, IonToolbar } from '@ionic/vue';
-import { defineComponent } from 'vue';
+import { IonContent, IonPage, IonTitle, IonToolbar, IonGrid, IonRow, IonButton } from '@ionic/vue';
+import { defineComponent, ref } from 'vue';
+import PurchaseOrder from '../types/purchaseOrder';
 
 export default defineComponent({
   name: 'Home',
   components: {
     IonContent,
-    IonHeader,
+ 
     IonPage,
     IonTitle,
-    IonToolbar
+    IonToolbar,
+    IonGrid,
+    IonRow,
+    IonButton
+  },
+  setup() {
+
+    const order = ref<PurchaseOrder[]>([
+      { purchaseOrderNumber: 'PO-0001', description: 'description one', date: new Date(), status: 'pending', action: 'reviewed' }
```

Comment:
```suggestion
     const orders = ref<PurchaseOrder[]>([]);
```

Reply:
please consider how you name your variables. instead of naming this `order`, it should be named `orders` since its an array of PurchaseOrder.

Reply:
done and noted